frontend/ui/translator.lua

```diff
@@ -526,6 +557,7 @@ function Translator:_showTranslation(text, target_lang, source_lang, from_highli
     end
     local output = {}
     local text_main = ""
+    local not_full_page = not full_page
```

Comment:
This feels ugly and prone to typo :)
And it means "detailed".
So, may be name it everywhere "detailed", and if the default should be true:
`if detailed == nil then detailed = true end`
and pass `false` when you don't want it.